ONSdigital/sdx-transform-cs
# Add implicit percentage values

## What? and Why?
There were a number of questions in the e-commerce section of the ecommerce 2019 survey where you are given multiple checkboxes.  If multiple ones are checked, they lead to a question that asks for a percentage breakdown, if only 1 is checked the breakdown is skipped.
The skipping is meant to imply that the option selected (and its corresponding percentage qcode) is 100%.  This wasn't done (or asked for originally) and this PR fixes that.

## How to test
- Spin up the service in SDX compose (only this branch needs to not be on master)
- Put through a 187.0001 with the following fields and qcodes removed:
   - 311, 312, 509, 510, 511.
- Check the PCK that was generated.  The following fields should have the following values
   - 310 10, 311 01, 312 01, 509 1000, 510 0, 511 0.

## Checklist
